opslevel.yml

```diff
@@ -0,0 +1,13 @@
+---
+version: 1
+repository:
+  name: rules
+  owner: dx_extensibility
+  lifecycle: deprecation
+  tier: Tier-3
```

Comment:
I wasn't quite sure which tier to use here. I picked Tier-3 because rules templates aren't really a customer impacting "service".

https://oktawiki.atlassian.net/wiki/spaces/ARCH/pages/2635727268/Decision+Failover+Expectations+by+Environment+Type+and+Service+Tier#Service-Tier

Reply:
Agreed. If there was a Tier-4 I would have recommended that